vulscan.nse

```diff
@@ -215,6 +215,17 @@ portrule = function(host, port)
 	end
 end
 
+function split(inputstr, sep)
+        if sep == nil then
+                sep = "%s"
+        end
+        local t={}
+        for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+                table.insert(t, str)
+        end
+        return t
+end
+
```

Comment:
add the split function, since it's throwing an error, this will resolve the undefined variable and cause the script to work properly when the .strsplit functions are replaced with .split later in the file